modular_skyrat/modules/new_meteors/readme.md

```diff
@@ -0,0 +1,16 @@
+https://github.com/Skyrat-SS13/Skyrat-tg/pull/18813
+
+## Title:
+MODULE ID: NEW_METEORS
```

Comment:
sometimes i wish we had sub-modules, i.e. a big "Events" folder where we put all this stuff... woe is me...